### PR TITLE
[Docs] Release Notes 2.0.0a9

### PR DESCRIPTION
**Pull request by aquamatthias**
See the rendered version here: https://github.com/someengineering/cloudkeeper/blob/mv/release_notes/release-notes.md

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
+
+# Release Notes 2.0.0a9 (Dec 9th, 2021)
+
+We are happy to announce another small increment towards our plan to a stable 2.0 version.
```
Comment (TheCatLady) [inline release-notes.md:4]:
```suggestion
We are very happy to announce that we are now another small step closer to a stable 2.0 release!
```

File: release-notes.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+
+The following 4 feature increments should be highlighted:
```
Comment (TheCatLady) [inline release-notes.md:6]:
```suggestion
Here are some highlights from this release:
```

File: release-notes.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+
+**The UI is now shipped as part of every release**<br/>
```
Comment (TheCatLady) [inline release-notes.md:8]:
```suggestion
**The UI is now shipped as part of every release**
```
We shouldn't need `<br/>` here?

File: release-notes.md [hunk 4]
```diff
@@ -1,3 +1,50 @@
...
+This is the first version that ships with our gorgeous UI.
+Try it out by downloading the latest version and accessing it via `https://path.to.cloudkeeper:8900/ui`.
+The main graph view has changed from 2D to 3D and shows Treemap charts (#457).
+
+**We now have a helm chart**<br/>
```
Comment (TheCatLady) [inline release-notes.md:13]:
```suggestion
**We now have a Helm chart**
```

File: release-notes.md [hunk 5]
```diff
@@ -1,3 +1,50 @@
...
+Thanks to a contribution from @yuval-k, we now have a helm chart (#428).
+With this chart in place, it is straightforward to deploy cloudkeeper in Kubernetes.
+Try it out yourself by following the [instructions](https://docs.some.engineering/getting_started/setup_kubernetes.html).
```
Comment (TheCatLady) [inline release-notes.md:16]:
```suggestion
With this chart, it is easier to deploy Cloudkeeper in Kubernetes.
Try it out yourself by following the [Kubernetes setup tutorial in our documentation](https://docs.some.engineering/getting_started/setup_kubernetes.html).
```

File: release-notes.md [hunk 6]
```diff
@@ -1,3 +1,50 @@
...
+
+**All cleanup plugins are now available**<br/>
+We needed to migrate all cleanup plugins to the 2.0 infrastructure.
+With this release, all cleanup plugins have been ported (#422) and (#439).
+
+**We integrated analytics sensors**<br/>
+You only care what you measure. We at Some Engineering believe, that it is important to know
+how Cloudkeeper is used and how we can improve it.
```
Comment (TheCatLady) [inline release-notes.md:24]:
```suggestion
At Some Engineering, we believe that it is important to know how Cloudkeeper is used, and thus how we can improve.
```

File: release-notes.md [hunk 7]
```diff
@@ -1,3 +1,50 @@
...
+That is the reason why we added analytics to our codebase.
+The data that we collect is anonymized with the pure focus of product insights.
+If this is a red flag for you, it is possible to opt-out via the command line flag.
+
+
+**Query language improvements**
+There are multiple bigger improvements here.
+It is now possible to define sub-queries (#412), that allow merging nodes with other nodes in the graph.
+
+Furthermore, the first step for a full-featured query template engine has been made with (#431).
```
Comment (TheCatLady) [inline release-notes.md:34]:
```suggestion
Additionally, the first step toward a full-featured query template engine has been implemented in #431.
```

File: release-notes.md [hunk 8]
```diff
@@ -1,3 +1,50 @@
...
+**We integrated analytics sensors**<br/>
+You only care what you measure. We at Some Engineering believe, that it is important to know
+how Cloudkeeper is used and how we can improve it.
+That is the reason why we added analytics to our codebase.
+The data that we collect is anonymized with the pure focus of product insights.
+If this is a red flag for you, it is possible to opt-out via the command line flag.
+
+
+**Query language improvements**
+There are multiple bigger improvements here.
```
Comment (TheCatLady) [inline release-notes.md:31]:
```suggestion
There are several significant improvements in this area.
```

File: release-notes.md [hunk 9]
```diff
@@ -1,3 +1,50 @@
...
+It is now possible to define sub-queries (#412), that allow merging nodes with other nodes in the graph.
+
```
Comment (TheCatLady) [inline release-notes.md:33]:
```suggestion
It is now possible to define sub-queries (#412) which allow merging nodes with other nodes in the graph.
```

**Review by TheCatLady — COMMENTED** [review]
Why aren't we publishing these release notes at https://github.com/someengineering/cloudkeeper/releases?

File: release-notes.md (hunk 3 above)
Comment (aquamatthias) [reply release-notes.md:8]: I wanted to make every highlighted feature standing out on one line.

**Pull request merged**